src/components/Header/Nav/ProductNavigation/Desktop/MegaMenu.module.css

```diff
@@ -0,0 +1,157 @@
+.megaMenuContainer {
+  z-index: 9999;
+  position: fixed;
+  top: 60px;
+  bottom: 0%;
+  left: 20px;
+  right: 20px;
+  display: none;
+  justify-content: center;
+}
+
+.wrapper {
+  max-width: 1360px;
+  width: 100%;
+  background-color: white;
+  height: fit-content;
+  border-radius: 0.5rem;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  overflow: hidden;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.wrapper::before {
```

Comment:
Not sure if this is the little Triangle on top o the menu, but that is not aligning properly. I tried on smaller and larger screens and the triangle does not fit properly. Maybe a simple solution would be to actually let that be part of the "button" and not the menu itself.

Let me know if I can help brainstorm